static/js/models/association.model.js

```diff
@@ -8,20 +8,26 @@ const Association = function (massociation) {
     this.hg19 = massociation.hg19;
     this.hg18 = massociation.hg18;
     this.hg17 = massociation.hg17;
+    this.trait = massociation.trait;
     this.gene = massociation.gene;
     this.raf = massociation.raf;
     this.riskAllele = massociation.riskAllele;
     this.pValue = massociation.pValue;
+    this.pValueAnnotation = massociation.pValueAnnotation;
     this.oddsRatio = massociation.oddsRatio;
     this.lowerCI = massociation.lowerCI;
     this.upperCI = massociation.upperCI;
-    this.pValueAnnotation = massociation.pValueAnnotation;
+    this.betaValue = massociation.betaValue;
+    this.betaUnit = massociation.betaUnit;
+    this.betaAnnotation = massociation.betaAnnotation;
+    this.ogValueTypes = massociation.ogValueTypes;
     this.sex = massociation.sex;
+    this.numAssociationsFiltered = massociation.numAssociationsFiltered; //this should be going away and should never be used from this model
```

Comment:
Do you want me to remove this column from the tables?

Reply:
We can remove this later